### ISSUE TITLE
margin and padding for View, Image and Text

### ISSUE THREAD

**Issue by raphamorim** [raphamorim/react-ape#118]
Implementation of margin and padding for existent elements

**Comment by Harshalyadav** [comment]
May, I take this issue?

**Comment by raphamorim** [comment]
Few free to take it @Harshalyadav but please start with one, breaking in three PRs: image, text, view (the order you take doesn’t matter)

**Comment by raphamorim** [comment]
@Harshalyadav I have broken into smaller issues (View, Text and Image). Later let me know which one want to work on, so I can assign the issue to you :)

**Comment by Harshalyadav** [comment]
Could you give me more info about this issue?

**Comment by raphamorim** [comment]
Yes, so the goal is to have the Style Props working for View. Both style props, margin and padding, follow React Native same directives and the work it works. Which is inspired by the way it works on CSS in Web browsers (https://blog.hubspot.com/website/css-margin-vs-padding).

- Margin style prop definition: https://reactnative.dev/docs/layout-props#margin
- Padding style prop definition: https://reactnative.dev/docs/layout-props#padding

**Comment by AAYUSH-droid** [comment]
hey u distributed this issue among #118 #119 #120?

**Comment by raphamorim** [comment]
yup @AAYUSH-droid

**Comment by Harshalyadav** [comment]
@raphamorim, Thank you.
